buildtest/cli/buildspec.py

```diff
@@ -973,14 +971,14 @@ def print_raw_filter_fields(self):
         method implements command ``buildtest buildspec find --filterfields``
         """
         for field in self.filter_fields:
-            console.print(field)
+            console.print(f"[{self.color}]{field}")
 
     def print_raw_format_fields(self):
         """This method prints the raw format fields available for buildspec cache. This
         method implements command ``buildtest buildspec find --formatfields``
         """
         for field in self.format_fields:
-            console.print(field)
+            console.print(f"[{self.color}]{field}")
```

Comment:
same here i think you can do `console.print(field,style=self.color)` since each field is a str type